doc/systest_basics.rst

```diff
@@ -0,0 +1,288 @@
+- **StarXGalaxyShear**: shear-shear cross correlation of galaxies and stars (shear-shear)
+- **StarXStarShear**: autocorrelation of the shapes of stars (shear-shear)
+- **StarXStarSize**: autocorrelation of the size residuals for stars, relative to PSF sizes
+  (scalar-scalar)
+- **GalaxyDensityCorrelation**: position autocorrelation of galaxies (point-point)
+- **StarDensityCorrelation**: position autocorrelation of stars (point-point)
+- **Rho1**: rho1 statistics (autocorrelation of residual star shapes)  (shear-shear)
+- **None**: creates a more flexible but less automatic class.  Instead of the usual call signature,
+  which just takes one or more data sets, the first argument to an object created this way must
+  be a TreeCorr correlation function type (such as ``gg`` or ``ng``).  This object also won't
+  have the ``.objects_list`` and ``.required_quantities`` described above.
+
+Calling these objects returns a formatted NumPy array of the correlation functions.  If the order
+of the data sets matters--eg for **GalaxyShear**--any data set that only needs positions should go
+first, and any data set with a scalar value you're using should go last.  For example:
+
+>>> bright_star_shear = stile.CorrelationFunctionSysTest('BrightStarShear')
+>>> results = bright_star_shear(bright_star_data, galaxy_data)
+
+The plotting method returns a :class:`matplotlib.figure.Figure` instance that can be saved
+by :func`.savefig` or further altered if you like.
+
+The estimators are different depending on the type.  Point-point estimates by default use
+the Landy-Szalay estimator:
+
+.. math::
+        \xi = (DD-2DR+RR)/RR
+
+and must include a random catalog.
+
+All shears in the following descriptions are the complex form in the frame aligned with the
+vector between the two points, that is, :math:`g = \gamma_t + i\gamma_x`.
+
+Point-shear estimates are equivalent to average tangential shear, returned as real
+:math:`\langle \gamma_t \rangle` and imaginary :math:`\langle \gamma_x \rangle`.  If random
+catalogs are given, they are used as random lenses, and
+:math:`\langle \gamma_t \rangle-\langle \gamma_{t, {\rm rand}} \rangle` is returned instead.
+
+Shear-shear correlation functions are :math:`\xi_+` and :math:`\xi_-`.  :math:`\xi_+` is,
+nominally, *g1* times *g2\**, and is given as both the real and imaginary components.
+:math:`\xi_{+,im}` should be consistent with 0 to within noise. :math:`\xi_-` on the other hand is
+*g1* times *g2* (not complex conjugate).  Similarly, :math:`\xi_{-,im}` should be 0.  (Note that
+*g1* and *g2* here are two *complex* shears :math:`g_{1t} + ig_{1x}` and :math:`g_{2t} + ig_{2x}`
+from the two catalogs, not the two components of a single shear in sky coordinates or chip frame.)
+
+Point-scalar (point-kappa) estimates are equivalent to <scalar>; scalar-shear estimates are
+equivalent to <scalar*Re(shear)> with a corresponding imaginary case; scalar-scalar estimates
```

Comment:
So perhaps say explicitly that in this case, the real (imaginary) case is the tangential (45 degree) shear of the 2nd sample about the positions defined by the 1st sample?